packages/http-client-java/generator/http-client-generator-test/src/main/java/com/_specs_/azure/core/lro/rpc/models/OperationState.java

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+package com._specs_.azure.core.lro.rpc.models;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.util.ExpandableStringEnum;
+import java.util.Collection;
+
+/**
+ * Enum describing allowed operation states.
+ */
+public final class OperationState extends ExpandableStringEnum<OperationState> {
```

Comment:
Caused by [case1](ttps://gist.github.com/haolingdong-msft/df3f25a048baf83d8fabf5206aacc7db#1-responses-header-schema-has-public-access-in-tcgc) and [case2](https://gist.github.com/haolingdong-msft/df3f25a048baf83d8fabf5206aacc7db#2-lro-initial-response-has-public-access-in-tcgc)